doc/classes/GeometryInstance3D.xml

```diff
@@ -25,7 +25,7 @@
 				Set the value of a shader uniform for this instance only ([url=$DOCS_URL/tutorials/shaders/shader_reference/shading_language.html#per-instance-uniforms]per-instance uniform[/url]). See also [method ShaderMaterial.set_shader_parameter] to assign a uniform on all instances using the same [ShaderMaterial].
 				[b]Note:[/b] For a shader uniform to be assignable on a per-instance basis, it [i]must[/i] be defined with [code]instance uniform ...[/code] rather than [code]uniform ...[/code] in the shader code.
 				[b]Note:[/b] [param name] is case-sensitive and must match the name of the uniform in the code exactly (not the capitalized name in the inspector).
-				[b]Note:[/b] Per-instance shader uniforms are currently only available in 3D, so there is no 2D equivalent of this method.
+				[b]Note:[/b] Per-instance shader uniforms are only available in 3D and 2D but not for Fog, Sky or Particles.
```

Comment:
Wouldn't it be better to specify Spatial and CanvasItem?

Reply:
```suggestion
				[b]Note:[/b] Per-instance shader uniforms are only available in Spatial and CanvasItem shaders, but not Fog, Sky or Particles shaders.
```

Reply:
The descriptions you wrote for the new methods in this PR correspond to the existing descriptions for the instance geometry methods, but are ever-so-slightly altered for the best.
For example, compared to `instance_geometry_get_shader_parameter_list`, `canvas_item_get_instance_shader_parameter_list` splits the description in two lines and omits the _"Equivalent to [method GeometryInstance3D.get_instance_shader_parameter]."_ because it does not make sense.
In hindsight though, there's no need to do this now.